Password Reset Section (#3443)
mattermost/docs: source/install/troubleshooting.rst

4. Replace ``{username}`` with the name of the user you'd like to promote to an admin.

> Password Resets

Reviewer: ```suggestion
Password resets
```

~~~~~~~~~~~~~~~

> Scenario #1: Internal users

Reviewer: ```suggestion
**Email Address or Username Sign In**
```
Changed this to match the content in the provided link.

> - Password reset request and emails are ONLY applicable to internal users (i.e users who are invited through invite link / email invite) based on documentation - https://docs.mattermost.com/help/getting-started/signing-in.html#email-address-or-username-sign-in.

Reviewer: ```suggestion
- If you sign in to Mattermost using an email address or username, you can send a password reset request by selecting **I forgot my password**. 
```

> - Since Mattermost Support do not have direct access to the server / instance the user has signed up to, we recommend users to get in touch directly to the System Administrator of the system

Reviewer: ```suggestion
As Mattermost Support do not have direct access to your Mattermost server/instance if you experience any problems, the next step is to contact your Mattermost System Admin.
```

> - If you are not sure who the System Administrator is, we suggest users to get in touch with the internal IT team for further help.

Reviewer: ```suggestion
If you are not sure who the System Admin is, contact the internal IT team at your organization for further help.
```

> Scenario #2 : AD / LDAP users

Reviewer: ```suggestion
**AD/LDAP users**
```

> - Password reset process should be done on the directory level instead of Mattermost UI since Mattermost pulls the information of the credentials from the LDAP directory.

Reviewer: ```suggestion
The password reset process is performed on the directory level as Mattermost pulls the credential information from the LDAP directory.
```

> - If the LDAP user has issues with the password, they should get in touch with the LDAP administrator for further help.

Reviewer: ```suggestion
If you experience issues with your password for your LDAP log in, contact your organization's LDAP administrator for further help.
```

> Scenario #3: SSO Users (OneLogin / Okta / etc)

Reviewer: ```suggestion
**SSO Users (OneLogin/Okta etc)**
```
Does this also include Google and GitLab? Or just Okta and OneLogin?

> - Password reset is to be done on the IdP provider side instead of Mattermost.

Reviewer: ```suggestion
The password reset process is completed on the IdP provider side, and not via the Mattermost System Console.
```

> SAML Issues

Reviewer: ```suggestion
SAML issues
```

4. Replace ``{username}`` with the name of the user you'd like to promote to an admin.

Password resets
~~~~~~~~~~~~~~~

**Email address or username sign in**

If you sign in to Mattermost using an email address or username, you can send a password reset request by selecting **I forgot my password**. 

> Mattermost Support do not have access to your Mattermost server/instance. If you experience any problems, the next step is to contact your Mattermost System Admin.

Reviewer 2: @justinegeffen can you confirm if this should be "Mattermost Support does not...." Not sure of the correct way as when I read it out loud, I can see both working.

Reviewer: Good point - I've reworded it. Let me know what you think? I think it reads better.

> Mattermost Support do not have access to your Mattermost server/instance. If you experience any problems, the next step is to contact your Mattermost System Admin.

Reviewer: ```suggestion
The Mattermost Support team does not have access to your Mattermost server/instance. If you experience any problems, the next step is to contact your Mattermost System Admin.
```

Reviewer 2: @justinegeffen yes, this reads better.

Author: Yes, sounds better. Thanks for the suggestion, @justinegeffen .

- If you are not sure who the System Administrator is, we suggest users to get in touch with the internal IT team for further help.

**AD/LDAP users**

- Password reset process should be done on the directory level instead of Mattermost UI since Mattermost pulls the information of the credentials from the LDAP directory.
- If the LDAP user has issues with the password, they should get in touch with the LDAP administrator for further help.

**SSO users (OneLogin/Okta etc)**

> 

Reviewer: Does this apply only to OneLogin and Okta?

Author: Yes, @justinegeffen . This applies to all SSO mentioned in the [documentation](https://docs.mattermost.com/deployment/sso-saml.html). The common ones that we notice are the first two.

Reviewer: Brilliant, thanks for clarifying. All looks good to me - thank you for putting this together, it's much appreciated!
